modules/apps/commerce/commerce-theme-speedwell/commerce-theme-speedwell-site-initializer/src/main/resources/com/liferay/commerce/theme/speedwell/site/initializer/internal/dependencies/journal_articles/SPEEDWELL_ACCOUNT.ftl

```diff
@@ -1,7 +1,7 @@
 <div class="account-dashboard">
 	<h2 class="account-dashboard__title">${PageTitle.getData()}</h2>
 
-	<a class="account-dashboard__pill" href="${NavigationItemLinkOrders.getFriendlyUrl()}">
+	<a class="account-dashboard__pill" href="${themeDisplay.getPathFriendlyURLPublic() + themeDisplay.getScopeGroup().getFriendlyURL() + "/my-orders"}">
```

Comment:
@gianmarcobrunialti assign "themeDisplay.getPathFriendlyURLPublic() + themeDisplay.getScopeGroup().getFriendlyURL()" to a varibale and then use it for all links